<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>9</y>
     <width>611</width>
     <height>361</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Navigator</string>
    </attribute>
    <widget class="QRadioButton" name="radioButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Cutting Tool Faces</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>190</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Cutting Tool Solids</string>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>80</y>
       <width>256</width>
       <height>101</height>
      </rect>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidget_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>210</y>
       <width>256</width>
       <height>111</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>80</y>
       <width>161</width>
       <height>101</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_3"/>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>210</y>
       <width>161</width>
       <height>111</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_4"/>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>300</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Show</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>PlatForm Hight</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_3">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>204</x>
       <y>20</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>374</x>
       <y>20</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>PlatForm Thickness</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_4">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>474</x>
       <y>20</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <attribute name="title">
     <string>Set Hub Angles</string>
    </attribute>
    <widget class="QWidget" name="gridLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>80</y>
       <width>231</width>
       <height>181</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2"/>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>120</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>A1 (degrees)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>66</x>
       <y>160</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>A2 (degrees)</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="a1_doubleSpinBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>120</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="decimals">
      <number>2</number>
     </property>
     <property name="minimum">
      <double>-100.000000000000000</double>
     </property>
     <property name="maximum">
      <double>100.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="a2_doubleSpinBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>160</y>
       <width>62</width>
       <height>22</height>
      </rect>
     </property>
     <property name="decimals">
      <number>2</number>
     </property>
     <property name="minimum">
      <double>-100.000000000000000</double>
     </property>
     <property name="maximum">
      <double>100.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
    </widget>
    <widget class="QCheckBox" name="setHubAngles_checkBox">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>10</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Set HubAngles</string>
     </property>
    </widget>
    <widget class="QLabel" name="a1Indegrees_label">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>120</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>120</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>rad</string>
     </property>
    </widget>
    <widget class="QLabel" name="a2Indegrees_label">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>160</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>160</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>rad</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_8">
    <attribute name="title">
     <string>Set Depth Cutting</string>
    </attribute>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>359</x>
       <y>80</y>
       <width>181</width>
       <height>171</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout"/>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>From Front Face</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>170</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>From Next To Front Face</string>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>231</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>180</y>
       <width>231</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QTableWidget" name="depthFromFrontFace_tableWidget">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>211</width>
       <height>111</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>Depth</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Hub Thickness</string>
      </property>
     </column>
    </widget>
    <widget class="QTableWidget" name="depthFromNextToFrontFace_tableWidget">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>190</y>
       <width>211</width>
       <height>131</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>Depth</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Hub Thickness</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="addDepthFromFront_pushButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>70</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QPushButton" name="removeDepthFromFront_pushButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>100</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addDepthFromNextToFront_pushButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>220</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QPushButton" name="removeDepthFromNextToFront_pushButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>250</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="setDepthCutting_checkBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>10</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Set Depth Cutting</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_7">
    <attribute name="title">
     <string>Set Fir Tree</string>
    </attribute>
    <widget class="QCheckBox" name="setFirTree_checkBox">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>10</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Set Fir Tree</string>
     </property>
    </widget>
    <widget class="QTabWidget" name="tabWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>611</width>
       <height>311</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>Set Position</string>
      </attribute>
      <widget class="QLabel" name="label_18">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>50</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>rad</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>50</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>A3 (degrees)</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="a3_doubleSpinBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>50</y>
         <width>62</width>
         <height>22</height>
        </rect>
       </property>
       <property name="decimals">
        <number>2</number>
       </property>
       <property name="minimum">
        <double>-100.000000000000000</double>
       </property>
       <property name="maximum">
        <double>100.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>70</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>A4 (degrees)</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_20">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>70</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>rad</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="a4_doubleSpinBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>70</y>
         <width>62</width>
         <height>22</height>
        </rect>
       </property>
       <property name="decimals">
        <number>2</number>
       </property>
       <property name="minimum">
        <double>-100.000000000000000</double>
       </property>
       <property name="maximum">
        <double>100.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="a3Indegrees_label">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>50</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
      <widget class="QLabel" name="a4Indegrees_label">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>70</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
      <widget class="QWidget" name="gridLayoutWidget_5">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>40</y>
         <width>161</width>
         <height>171</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_5"/>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Hub Thickness</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="firTreeHubThickness_doubleSpinBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>100</y>
         <width>62</width>
         <height>22</height>
        </rect>
       </property>
       <property name="decimals">
        <number>4</number>
       </property>
       <property name="minimum">
        <double>-99.999899999999997</double>
       </property>
       <property name="maximum">
        <double>99.999899999999997</double>
       </property>
       <property name="singleStep">
        <double>0.000100000000000</double>
       </property>
      </widget>
      <widget class="Line" name="line_5">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>50</y>
         <width>20</width>
         <height>71</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_6">
      <attribute name="title">
       <string>Set Shape</string>
      </attribute>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>151</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Fir Tree Bottom Fillet Radius (R)</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>151</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Fir Tree Upper Fillet Radius (r)</string>
       </property>
      </widget>
      <widget class="QPushButton" name="removeLobe_pushButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>260</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="fiTreeBottomFilletRadius_doubleSpinBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>10</y>
         <width>62</width>
         <height>22</height>
        </rect>
       </property>
       <property name="decimals">
        <number>4</number>
       </property>
       <property name="minimum">
        <double>-99.999899999999997</double>
       </property>
       <property name="maximum">
        <double>99.999899999999997</double>
       </property>
       <property name="singleStep">
        <double>0.000100000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>80</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Neck Bottom Width (L0)</string>
       </property>
      </widget>
      <widget class="QPushButton" name="addLobe_pushButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>260</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="fiTreeUpperFilletRadius_doubleSpinBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>30</y>
         <width>62</width>
         <height>22</height>
        </rect>
       </property>
       <property name="decimals">
        <number>4</number>
       </property>
       <property name="minimum">
        <double>-99.999899999999997</double>
       </property>
       <property name="maximum">
        <double>99.999899999999997</double>
       </property>
       <property name="singleStep">
        <double>0.000100000000000</double>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>60</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Neck Height (nh)</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="neckBottomWidth_doubleSpinBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>80</y>
         <width>62</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QTableWidget" name="lobeProperties_tableWidget">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>150</y>
         <width>611</width>
         <height>101</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777209</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="lineWidth">
        <number>0</number>
       </property>
       <property name="autoScrollMargin">
        <number>5</number>
       </property>
       <column>
        <property name="text">
         <string>Load Length</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Load Angle</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Lobe Thickness</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Widths (L)</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Convex Radius</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Concave Radius</string>
        </property>
       </column>
      </widget>
      <widget class="QDoubleSpinBox" name="neckHeight_doubleSpinBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>60</y>
         <width>62</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="Line" name="line_3">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>10</y>
         <width>20</width>
         <height>91</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="QWidget" name="gridLayoutWidget_6">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>0</y>
         <width>160</width>
         <height>141</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_6"/>
      </widget>
     </widget>
    </widget>
    <widget class="Line" name="line_4">
     <property name="geometry">
      <rect>
       <x>-3</x>
       <y>20</y>
       <width>611</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Set Hub Bottom line</string>
    </attribute>
    <widget class="QCheckBox" name="checkBox_4">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>10</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Set Hub Bottom line</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Set Cutting Angle</string>
    </attribute>
    <widget class="QCheckBox" name="checkBox_5">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>10</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Set Cutting Angle</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>349</x>
     <y>380</y>
     <width>239</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="finishButton">
      <property name="text">
       <string>Finish</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Apply</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="occFirTree.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
